compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion.hs

```diff
@@ -751,6 +756,9 @@ convertExpr env0 e = do
             mkFieldProj i (name, _typ) = (mkField ("_" <> T.pack (show i)), ETupleProj name (EVar varV1))
     go env (VarIs "primitive") (LType (isStrLitTy -> Just y) : LType t : args)
         = fmap (, args) $ convertPrim (envLfVersion env) (unpackFS y) <$> convertType env t
+    go env (VarIs "external") (LType (isStrLitTy -> Just y) : LType t : args) = do
```

Comment:
Please check the module name and maybe even the package name, we should try to reduce the functions that just do string matching, not add more.

Reply:
done.